ocs_ci/ocs/pgsql.py

```diff
@@ -14,7 +14,7 @@
 from ocs_ci.ocs.resources.ocs import OCS
 from ocs_ci.ocs import constants
 from subprocess import CalledProcessError
-from ocs_ci.ocs.resources.pod import get_all_pods, get_pod_obj, get_operator_pods
+from ocs_ci.ocs.resources.pod import get_all_pods, get_pod_obj, get_operator_pods, get_file_path
```

Comment:
Please break the line, it has exceeded 79 char

Reply:
done